packages/gatsby/lib/query-runner/compiler-helpers.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import fs from "fs"
+import path from "path"
+import { FileIRParser, FileWriter, IRTransforms } from "relay-compiler"
+import { buildASTSchema, parse } from "graphql"
+import { store } from "../redux"
```

Comment:
Eh... I dunno. This isn't part of webpack at all so I don't see why we should use webpack.